config.go

```diff
@@ -0,0 +1,162 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/bitrise-io/go-steputils/stepconf"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/go-utils/pathutil"
+)
+
+// Configs stores the step's inputs
+type Configs struct {
+	JSONKeyPath             stepconf.Secret `env:"service_account_json_key_path,required"`
+	PackageName             string          `env:"package_name,required"`
+	AppPath                 string          `env:"app_path,required"`
+	ExpansionfilePath       string          `env:"expansionfile_path"`
+	Track                   string          `env:"track,required"`
```

Comment:
user fraction is a required step input, in config level I defined `opt`, so it can get values from the give list only: https://github.com/bitrise-io/go-steputils/blob/master/stepconf/stepconf.go#L138

Reply:
Indeed, thanks.